testgrid/config.yaml

```diff
@@ -4165,6 +4168,11 @@ dashboards:
   - name: pr-verify
     test_group_name: pull-cluster-api-provider-docker-verify
 
+- name: sig-cluster-lifecycle-cluster-api-bootstrap-provider-kubeadm
```

Comment:
You only need this line in the testgrid config. The rest can now be configured with annotations on the ProwJob which is the preferred way of doing things. See these docs: https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md#prow-job-configuration

Reply:
ack, i saw the annotations but haven't sent PRs with them yet.
let me see if i can get it right here.